### PR TITLE
Updating WebGLRenderingContext for node

### PR DESCRIPTION
**Pull request by michaeljherrmann**
##### Description of change

I'm following up on [this PR](https://github.com/pixijs/pixijs/pull/8643) as the ideal fix we wanted required some updates to some dependencies. I've got those updates merged in [gl](https://github.com/stackgl/headless-gl/pull/236) and for it's [types](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62647).

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] Lint process passed (`npm run lint`)
- [ ] Tests passed (`npm run test`)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -144,11 +144,11 @@
         "@pixi/spritesheet": "7.1.0-alpha",
```
Comment (michaeljherrmann) [inline package-lock.json:144]: I'm not sure if I regenerated the lock file correctly? seems to have more changes than I expected. I just ran `npm install` in the top level after updating the `package.json` for node

**Comment by michaeljherrmann** [comment]
I'm guessing @Zyie should probable review this?

**Comment by michaeljherrmann** [comment]
I think I need to fix some more typings still in `DefinitelyTyped`

**Review by bigtimebuddy — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (SuperSodaSea) [reply package-lock.json:144]: I tried `npm i gl@6.0.1 @types/gl@6.0.1` in `bundles/pixi.js-node` and got the same result, so I think it is OK. Possibly it is caused by the dependency change of `gl`.

Comment (SuperSodaSea) [reply package-lock.json:144]: @bigtimebuddy by the way, should we have a `npm audit fix` for 7.1.0?

Comment (bigtimebuddy) [reply package-lock.json:144]: Yeah, we tried an audit awhile ago (#8480), but kept being blocked by various things.  I'm open to doing this again.

**Comment by SuperSodaSea** [comment]
By the way, it seems that the extension interfaces like `STACKGL_resize_drawingbuffer` and `STACKGL_destroy_context` are not exported in `@types/gl`? I am trying to use `STACKGL_resize_drawingbuffer` and got `Module '"gl"' has no exported member 'STACKGL_resize_drawingbuffer'`. If they can be exported we can have a better typing.

**Comment by SuperSodaSea** [comment]
@michaeljherrmann Now that my PR for `@types/gl` is merged, would you like to bump it to 6.0.2?

**Comment by michaeljherrmann** [comment]
> @michaeljherrmann Now that my PR for `@types/gl` is merged, would you like to bump it to 6.0.2?

@SuperSodaSea done!

**Review by SuperSodaSea — APPROVED** [review]
Thanks!

**Pull request merged**